web/src/features/charts/tooltips/PriceChartTooltip.tsx

```diff
@@ -1,14 +1,14 @@
 import getSymbolFromCurrency from 'currency-symbol-map';
 import { useAtom } from 'jotai';
 import { useTranslation } from 'react-i18next';
-import { timeAverageAtom } from 'utils/state/atoms';
+import { timeRangeAtom } from 'utils/state/atoms';
 
 import { convertPrice } from '../bar-breakdown/utils';
 import { InnerAreaGraphTooltipProps } from '../types';
 import AreaGraphToolTipHeader from './AreaGraphTooltipHeader';
 
 export default function PriceChartTooltip({ zoneDetail }: InnerAreaGraphTooltipProps) {
-  const [timeAverage] = useAtom(timeAverageAtom);
+  const [timeRange] = useAtom(timeRangeAtom);
```

Comment:
I didn't know there was so many of these useAtom calls that should be useAtomValue instead...

I'll create a follow up PR for that.